diesel/src/macros/mod.rs

```diff
@@ -989,6 +989,17 @@ macro_rules! __diesel_table_query_source_impl {
 ///
 /// assert_eq!(implicit_on_clause_sql, explicit_on_clause_sql);
 /// # }
+///
+/// In the example above, in line `joinable!(posts -> users (user_id));` specifies the relation of
+/// the tables and the ON clause in the following way:
+///
+/// `table_with_foreighn_key -> table_that_foreign_key_points_to (foreign_key)`
```

Comment:
What do you think about calling this `child_table` and `parent_table`, and then specifying that the child table is the one with the foreign key, and the parent table is the one with the primary key that the child table points to? (This is how the association docs lay it out)

Reply:
Yes that sound better, and less confusing. I took a look at the [associations docs](http://docs.diesel.rs/diesel/associations/index.html), do you think we could link or use those definitions of User and Post tables?